yb-voyager/src/queryparser/query_parser.go

```diff
@@ -77,13 +78,13 @@ func IsViewObject(parseTree *pg_query.ParseResult) bool {
 }
 
 func IsMviewObject(parseTree *pg_query.ParseResult) bool {
-	createAsNode, isCreateAsStmt := parseTree.Stmts[0].Stmt.Node.(*pg_query.Node_CreateTableAsStmt) //for MVIEW case 
+	createAsNode, isCreateAsStmt := parseTree.Stmts[0].Stmt.Node.(*pg_query.Node_CreateTableAsStmt) //for MVIEW case
```

Comment:
consider defining helper functions for - `parseTree.Stmts[0].Stmt.Node.(*pg_query.Node_CreateTableAsStmt)`